assets/js/index.js

```diff
@@ -0,0 +1,112 @@
+// variables
+const weekDay = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+const startTime = 10;
+const endTime = 20;
+let times = [];
```

Comment:
An array variable should generally always be declared as `const` (unless you are planning on reassigning it to a different data type, like a number or a string, at some point). Even though it may seem like we are changing the array by adding into it, the variable name is still pointing to that same array. Think of it as a box we are putting stuff in - we can still change what is inside even though the actual box doesn't change.